### PR TITLE
[Spec] Binary process management

### PR DESCRIPTION
**Pull request by divyenduz**
Open [**Rendered Spec**](https://github.com/prisma/specs/tree/binary-process-management/binaries#binary-process-management) 📄
Addresses https://github.com/prisma/specs/issues/1

- [x] Binary process management (mainly in Photon)
- [x] Spawning
- [x] Execution and connection handling
- [x] Stopping and connection release

If there is anything significantly different
- [x] Migration engine binary process management

### CODE REVIEW

**Comment by divyenduz** [comment]
This is ready to be merged, please let me know if any changes are needed 🙏

**Comment by schickling** [comment]
@matthewmueller isn't this obsolete now based on [your spec on master](https://github.com/prisma/specs/tree/master/binaries)?

**Comment by divyenduz** [comment]
@matthewmueller This is merged into the consolidated binary spec now. Ready to be merged from my end. Feedback please.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Technical specification of Prisma
 
-- [Binary workflows](/binary-workflows)
+- [Binaries](/binaries)
```
Comment (divyenduz) [inline README.md]: Also fixed this link :)

**Pull request merged**